lib/h3core.js

```diff
@@ -69,7 +69,9 @@ export const UNITS = {
     m: 'm',
     km: 'km',
     m2: 'm2',
-    km2: 'km2'
+    km2: 'km2',
```

Comment:
nit: this is an odd grouping of types, seems like km/km2 should be next to each other, or all the area ones should be grouped together

Reply:
updated